tempesta_fw/http_sess.c

```diff
@@ -720,6 +725,18 @@ tfw_http_sticky_secret_cfg(TfwCfgSpec *cs, TfwCfgEntry *ce)
 	return 0;
 }
 
+static int
+tfw_http_sticky_sess_lifetime_cfg(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	int r = tfw_cfg_set_int(cs, ce);
+
+	/* @sess_lifetime value of 0 means unlimited. */
+	if (!r && !tfw_cfg_sticky.sess_lifetime)
```

Comment:
`0` in terms of `sess_lifetime` means `unlimited`, but setting actually zero to the parameter makes session to be removed right after it was created. As discussed before, `UINT_MAX` value should represent `unlimited` value for the code, but for the user it should remain as `0`.

I can rewrite condition as follows if you find that more meaningful:
```
if (!r && !*(cs->dest))
    *(cs->dest) = UINT_MAX;
```